aspnetcore/data/ef-mvc/complex-data-model.md

```diff
@@ -85,7 +85,7 @@ Suppose you want to ensure that users don't enter more than 50 characters for a
 The `StringLength` attribute won't prevent a user from entering white space for a name. You can use the `RegularExpression` attribute to apply restrictions to the input. For example, the following code requires the first character to be upper case and the remaining characters to be alphabetical:
 
 ```csharp
-[RegularExpression(@"^[A-Z]+[a-zA-Z""'\s-]*$")]
+[RegularExpression(@"^[A-Z]+[a-zA-Z]*$")]
```

Comment:
These inline versions are explained right above, where it confirms that it should just be what I've changed it to, without the special characters.